lib/docs/core/compat_tables.rb

```diff
@@ -0,0 +1,165 @@
+module CompatTables
+  require 'json'
+
+  TRANSLATEPATH = {
+    'element' => 'elements'
+  }
+
+  BROWSERS = {
+    'chrome' => 'Chrome',
+    'edge' => 'Edge',
+    'firefox' => 'Firefox',
+    'ie' => 'Internet Explorer',
+    'opera' => 'Opera',
+    'safari' => 'Safari',
+    'webview_android' => 'WebView Android',
+    'chrome_android' => 'Chrome Android',
+    'firefox_android' => 'Firefox for Android',
+    'opera_android' => 'Opera Android',
+    'safari_ios' => 'Safari on IOS',
+    'samsunginternet_android' => 'Samsung Internet'
+  }
+
+  GLOBALFILES = ['global_attributes']
+
+  def generateCompatibilityTable(doc, slug, documentation)
```

Comment:
```suggestion
  def generate_compatibility_table(doc, slug, documentation)
```

Ruby seems to follow the snake_case convention for function names.